messages/teleporter/utils.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	receiveMessageGasLimitBufferAmount = 100_000
+	receiveMessageGasLimitBufferAmount = 100_000_0
```

Comment:
Was there a reason this needed to be increased? 

1,000,000 extra gas is a lot, so ideally we don't have to increase it this much. Either way, lets format the number as `1_000_000` to make it easier to read.

Reply:
I was taking a guess as to why the e2e_test failed, and if it's running out of gas. Removed.